UdpDebugLog/src/SphereUdpLogSender/udplog.c

```diff
@@ -0,0 +1,84 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved.
+   Licensed under the MIT License. */
+
+#include "udplog.h"
+#include <stdbool.h>
+#include <stdint.h>
+
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+#include <unistd.h>
+
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <stdarg.h>
+
+#define PORT 1824
+
+static int 	sock=-1;
+
+static struct sockaddr_in broadcast_addr;
+static socklen_t addr_len;
+static int ret;
+static char sock_buffer[2048];
```

Comment:
To be safe with UDP you can only send 512 byte packets or smaller.  In practice you're usually safe up to the MTU, which is 1500 on many systems, but 2048 isn't likely to work.